lib/database/index.ts

```diff
@@ -14,7 +14,12 @@ import { Construct } from "constructs";
 import { CustomLambdaFunctionProps } from "../utils";
 
 const instanceSizes: Record<string, number> = require("./instance-memory.json");
-const DEFAULT_PGSTAC_VERSION = "0.7.10";
+const DEFAULT_PGSTAC_VERSION = "0.8.4";
```

Comment:
If we do that, we need to change the `pypgstac` pin in the ingestor https://github.com/developmentseed/eoapi-cdk/blob/a5e398e596fcb80398db10cfdb87aad9e20673bf/lib/ingestor-api/runtime/requirements.txt#L8